CI/vale/vale_styles/Vocab/Besu/accept.txt

```diff
@@ -36,6 +36,7 @@ Dev
 [dD]eauthoriz(e|ing)
 [eE]ncryptor(s)?
 [eE]node(s)?
+enum
```

Comment:
Not fond of having code keywords in this list. As it is in a table where the other types are "Quantity", "Data" or "Array", it would be more consistent to have "Enumeration" instead.

Reply:
Fixed